src/matrix_eqn.jl

```diff
@@ -168,3 +168,48 @@ function solve_discrete_riccati(A::ScalarOrArray, B::ScalarOrArray,
 
     return H0 + gamma .* I  # Return X
 end
+
+"""
+Simple method to return an element Z in the Riccati equation solver whose type is Float64 (to be accepted by the cond() function)
+
+##### Arguments
+
+- `BB::Float64` : result of B' * B
+- `gamma::Float64` : parameter in the Riccati equation solver
+- `R::Float64`
+
+##### Returns
+- `::Float64` : element Z in the Riccati equation solver
+
+"""
+getZ(R::Float64, gamma::Float64, BB::Float64) = R + gamma * BB
+
+"""
+Simple method to return an element Z in the Riccati equation solver whose type is Float64 (to be accepted by the cond() function)
+
+##### Arguments
+
+- `BB::Array{Float64, 1}` : result of B' * B
+- `gamma::Float64` : parameter in the Riccati equation solver
+- `R::Float64`
+
+##### Returns
+- `::Float64` : element Z in the Riccati equation solver
+
+"""
+getZ(R::Float64, gamma::Float64, BB::Array{Float64, 1}) = R + gamma * BB[1]
```

Comment:
We could, but I think there is no way it has more than 1 element, as it comes from `B' * B` - where `B` is either a `Vector` or `Matrix`